src/thinkspace/api/thinkspace-team/app/models/thinkspace/team/team_set.rb

```diff
@@ -89,6 +89,11 @@ def unlocked_teams
         thinkspace_team_teams.where.not(state: Thinkspace::Team::Team.state_locked)
       end
 
+      # ###
+      # ### Abstract
+      # ###
```

Comment:
This is pretty close to what our new comment headers should look like with some minor changes.

`# ###` will convert any of the following text to a H3 which is fine if thats what you want the `Abstract` to be, but the wrapping `# ###` with no text will just appear as redudant hash marks in the source portion.

Just having the `# ### Abstract` will suffice but if you wish to append and prepend extra room just use a single `#`

```
#
# ### Abstract
#
```

or ideally just use the single line